components/tikv_alloc/src/lib.rs

```diff
@@ -82,6 +82,7 @@
 #[macro_use]
 extern crate log;
 
+#[cfg(feature = "jemalloc")]
 #[macro_use]
 extern crate lazy_static;
```

Comment:
why need this?

Reply:
A latest update in master bring a `unused macro_use` warning.